GRAPHICS/SetScaleformMovieToUseLargeRt.md

```diff
@@ -0,0 +1,16 @@
+---
+ns: GRAPHICS
+aliases: ["0x32F34FF7F617643B"]
+---
+## SET_SCALEFORM_MOVIE_TO_USE_LARGE_RT
+
+```c
+// 0x32F34FF7F617643B
+void SET_SCALEFORM_MOVIE_TO_USE_LARGE_RT(int scaleformHandle, cs_type(Any) BOOL toggle);
+```
+
+Configures a Scaleform movie to render to a large render target (1280x720), which is useful for ensuring higher quality and clarity in certain display scenarios. Such as displaying the name of an organization (CEO Office) in a visually impactful way.
+
+## Parameters
+* **scaleformHandle**: The handle of the Scaleform to be used.
```

Comment:
This should be named `scaleformMovieId` since it's the scaleform movie id.

Reply:
Should be good